src/libs/actions/IOU.ts

```diff
@@ -9251,6 +9253,7 @@ function approveMoneyRequest(expenseReport: OnyxEntry<OnyxTypes.Report>, full?:
 
     const optimisticNextStep = buildNextStep(expenseReport, predictedNextStatus);
     const chatReport = getReportOrDraftReport(expenseReport.chatReportID);
+    const currentSearchQueryJSON = getCurrentSearchQueryJSON();
```

Comment:
The `snapshot` should already have the has in it. So no need to call this again here.

Reply:
Oh you're right. I could swear we had that key there 😅